doc/integrator/fulltext_search.rst

```diff
@@ -0,0 +1,257 @@
+.. _integrator_fulltext_search:
+
+Full-text search
+================
+
+Enabling the *full-text search* feature involves adding a ``FullTextSearch``
+plugin to the viewer, and creating and populating a dedicated table in the
+database.
+
+The FullTextSearch plugin
+-------------------------
+
+The viewer should include a ``FullTextSearch`` plugin for the *text search*
+feature to be available in the user interface.
+
+See the `FullTextSearch API doc
+<http://docs.camptocamp.net/cgxp/2.3/lib/plugins/FullTextSearch.html>`_ for the
```

Comment:
2.3 should be generated, cgxp is legacy

Reply:
Right, moving the cgxp part to legacy_cgxp.rst